docs/newbs_getting_started.md

```diff
@@ -120,7 +120,7 @@ NOTE: remember to follow the instructions printed at the end of installation (us
 
 ### ** Windows **
 
-After installing QMK you can set it up with this command:
+After installing QMK you can set it up with this command in the QMK MSYS terminal:
```

Comment:
```suggestion
After installing QMK, open QMK MSYS and run the following command:
```
Same for the macOS section below:
```suggestion
After installing QMK, open Terminal and run the following command:
```
And Linux:
```suggestion
After installing QMK, open your preferred terminal app and run the following command:
```

Reply:
Added these recommendations.